lib/ui/src/modules/ui/components/theme.js

```diff
@@ -1,7 +1,7 @@
 export const baseFonts = {
   fontFamily: `
-    -apple-system, ".SFNSText-Regular", "San Francisco", "Roboto",
-    "Segoe UI", "Helvetica Neue", "Lucida Grande", sans-serif
+    -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu",
+    "Cantarell", "Fira Sans", "Droid Sans", "Arial", sans-serif
```

Comment:
Well if the alternative is to have a font that jumps when going from normal to bold, the unfamiliarity is a lesser evil